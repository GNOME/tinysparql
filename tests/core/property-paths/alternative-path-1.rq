select foaf:name(?u) foaf:name(?a) { ?u a foaf:Person . ?u foaf:knows|foaf:member ?a } order by foaf:name(?u) foaf:name(?a)