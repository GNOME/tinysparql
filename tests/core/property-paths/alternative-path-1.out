"Alice"	"Bob"
"Alice"	"Foo"
"Bob"	"Alice"
"Bob"	"Foo"
"Eve"	"Fred"
